src/storage/v2/storage.hpp

```diff
@@ -402,6 +414,12 @@ class Storage final {
 
   ConstraintsInfo ListAllConstraints() const;
 
+  bool CreateSchema(LabelId primary_label, std::vector<SchemaProperty> &schemas_types);
```

Comment:
This is not implemented. Same as `DeleteSchema` below. Is this intended?

Reply:
Yes, it is intentional, it will be implemented in the next PR, since it will be used then

Reply:
Then `vector` should be const? Anyway, leave it for now. On the next PR should be ok!